### PR TITLE
feat(website): logo is not displayed in Navbar [Safari] #334

### LINKED ISSUE #334: Logo is not displayed in Navbar [Safari]
Problem refers to mobile and desktop

### PR DESCRIPTION
**Pull request by KrystianKjjk**
Closes #334 

Modified Navbar a bit more than only logo, because in some cases, couldn't set width to 40px (depends on the screen logo had width between 34-40px)

### CODE REVIEW

**Review by wierzbka10 — COMMENTED** [review]
I haven't safari so I can't check it, but I think that navabar height is good.

File: packages/website/src/components/Footer/Footer.data.tsx [hunk 1]
```diff
@@ -1,4 +1,5 @@
 import React, { ReactElement } from 'react';
+import { pageNavigation } from 'src/pageNavigation';
```
Comment (KonradSzwarc) [inline packages/website/src/components/Footer/Footer.data.tsx:2]: 💭 It's not defined in TS paths so it should be imported in a relative way

File: packages/website/src/components/Navbar/BaseNavbar.tsx [hunk 2]
```diff
@@ -1,12 +1,13 @@
 import React from 'react';
 import Link from 'next/link';
 import { useRouter } from 'next/router';
+import { pageNavigation } from 'src/pageNavigation';
```
Comment (KonradSzwarc) [inline packages/website/src/components/Navbar/BaseNavbar.tsx:4]: 💭 It's not defined in TS paths so it should be imported in a relative way

File: packages/website/src/components/Navbar/BaseNavbar.tsx [hunk 3]
```diff
@@ -19,37 +20,36 @@ import { externalLinkBaseProps } from '../ExternalLink';
...
-  {
-    text: 'FAQ',
-    destinationLink: '/faq',
-  },
-  {
-    text: 'Kontakt',
-    destinationLink: '/kontakt',
-  },
-];
+const BecomePartOfCourseButtons = () => {
```
Comment (KonradSzwarc) [inline packages/website/src/components/Navbar/BaseNavbar.tsx:23]: 💭 I think `NavbarButtons` could be enough

**Review by KonradSzwarc — APPROVED** [review]
Visually looks great! I only added a few code-related suggestions.

**Review by hoolek77 — APPROVED** [review]

**Pull request merged**